intro_to_python/intro_to_python.md

```diff
@@ -2,21 +2,22 @@
 
 author:   Meredith Lee
 email:    leemc@chop.edu
-version:  1.0.0
+version:  2.0.0
+module_template_version: 2.0.0
 language: en
 narrator: UK English Female
 title: Introduction to Python
-comment:  This module introduces the Python programming language, explores why Python is useful in research, and demonstrates how to write Python code in a Jupyter notebook (and why you might want to).
-long_description: Python is a versatile programming language that is frequently used for data analysis, machine learning, web development, and more. If are interested in learning about any of these topics and are new to programming, are joining a team where Python is used, or have just heard about Python and are looking for a general introduction to the language, this module is a good place to start. This is appropriate for someone at the beginner level.
-
+comment:  This module introduces the Python programming language, explores why Python is useful in research, and describes how to download Python and Jupyter.
+long_description: Python is a versatile programming language that is frequently used for data analysis, machine learning, web development, and more. If you are interested in learning about any of these topics and are new to programming, are joining a team where Python is used, or have just heard about Python and are looking for a general introduction to the language, this module is a good place to start. This is appropriate for someone at the beginner level.
```

Comment:
```suggestion
long_description: Python is a versatile programming language that is frequently used for data analysis, machine learning, web development, and more. If you are interested in using Python (or even just trying it out), and are looking for how to get set up, this module is a good place to start. This is appropriate for someone at the beginner level, including those with no prior knowledge of or experience with Python.
```

suggestion to increase specificity of what exactly is covered in this module, since this description was developed for the bigger previous module. (e.g., this version of the module doesn't really provide a general intro to the language as much as it just discusses the "why" and then gets learners set up to use it). feel free to totally change or reject changing it at all.

Reply:
I like this suggestion, it's a bit more targeted to the module as it is now.